corehq/apps/export/static/export/js/utils.js

```diff
@@ -1,4 +1,16 @@
-hqDefine('export/js/utils.js', function () {
+if (!String.prototype.endsWith) {
```

Comment:
Out of curiosity, is there indeed an implementation of js that we support that doesn't have `endsWith` available?